MapleServer2/Commands/Game/PlayerCommands.cs

```diff
@@ -127,4 +132,75 @@ public override void Execute(GameCommandTrigger trigger)
             trigger.Session.SendNotice("Level must be a number or more than 0.");
         }
     }
+
+    public class SkillCommand : InGameCommand
+    {
+        public SkillCommand()
+        {
+            Aliases = new[]
+{
+                "skill"
+            };
+            Description = "Cast an specific skill.";
+            AddParameter<int>("id", "ID of the skill.");
+            AddParameter<short>("level", "The level of the skill.");
+        }
+
+        public override void Execute(GameCommandTrigger trigger)
+        {
+            int id = trigger.Get<int>("id");
+            if (id == 0)
+            {
+                trigger.Session.SendNotice($"Skill with id: {id} is not defined.");
+                return;
+            }
+
+            SkillCast skillCast = new SkillCast(trigger.Get<int>("id"), 1, GuidGenerator.Long(), 1);
+            CoordF empty = CoordF.From(0, 0, 0);
+            IFieldObject<Player> player = trigger.Session.FieldPlayer;
+
+            trigger.Session.FieldManager.BroadcastPacket(SkillUsePacket.SkillUse(skillCast, player.Coord, empty, empty, player.ObjectId));
+        }
+    }
+
+    public class BuffCommand : InGameCommand
+    {
+        public BuffCommand()
+        {
+            Aliases = new[]
+{
+                "buff"
+            };
+            Description = "Level up all the skills available.";
+            AddParameter<int>("id", "ID of the status.");
```

Comment:
Unrelated to this PR but I think parameters should be an array on the Command class, and then you initialize it like this: 
```
Parameters = new [] {
    Parameter<int>("id", "ID"),
    Parameter<int>("id", "ID")
}
```

Reply:
Thats a good idea!